bpf/lib/common.h

```diff
@@ -336,25 +336,29 @@ struct proxy4_tbl_key {
 	__u16 dport; /* dport must be in front of sport, loaded with 4 bytes read */
 	__u16 sport;
 	__u8 nexthdr;
+	__u8 pad;
```

Comment:
Yes, I think we have to discuss this in some more details. The reason I've left out the destination IP is not necessarily an optimization but the fact that we may not know the real destination IP that the packet will carry when it is finally delivered into the socket of the proxy. There may be additional DNAT rules in the stack. Maybe this is being overly cautious and we can ignore this corner case.

Are you OK to discuss this separately outside of this PR?